src/LodeRunner.Data/Services/TestRunService.cs

```diff
@@ -74,5 +75,25 @@ public new async Task<HttpStatusCode> Delete(string id)
                 return HttpStatusCode.InternalServerError;
             }
         }
+
+        /// <summary>
+        /// Gets all available TestRuns for the given client id.
+        /// </summary>
+        /// <param name="clientId">The client id.</param>
+        /// <returns>
+        /// List of TestRuns to run on client.
+        /// </returns>
+        public async Task<IEnumerable<TestRun>> GetAvailableTestRunsByClientIdAsync(string clientId)
+        {
+            string sql = $"SELECT * FROM e WHERE e.entityType='TestRun' ";
+            sql += $"and array_contains(e.loadClients, {{ \"id\": \"{clientId}\"}}, true) ";
+
+            // TODO: Update and add later once LoadResults schema is defined
+            // sql + = $"and NOT array_contains(e.ClientResults, {{\"id\": \"{clientId}\"}}, true)";
+            sql += $"and (NOT IS_DEFINED(e.completedTime) or IS_NULL(e.completedTime)) ";
+            sql += $"ORDER BY e.startTime ASC";
```

Comment:
FYI - This will currently consider a TestRun completed if completedTime is set to an empty string ("").  I can update this query if completedTime = "" is a valid/expected scenario to indicate that a TestRun has not been completed.

Reply:
@aflinchb, let's update it.  CompletedTime with a valid datetime later than the StartTime is completed.  If CompletedTime is undefined, null, or an empty string then it should be viewed as not complete.